_includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md

```diff
@@ -28,6 +28,7 @@ Make sure that the following modules have been installed:
 |------------------------------------|--------------------------------------------------------|
 | ProductOfferServicePoint           | vendor/spryker/product-offer-service-point             |
 | ProductOfferServicePointDataImport | vendor/spryker/product-offer-service-point-data-import |
+| ProductOfferServicePointStorage    | vendor/spryker/product-offer-service-point-storage     |
```

Comment:
this module is mapped to another feature
see https://release.spryker.com/features/view/145